docker-compose.yml

```diff
@@ -1,18 +1,38 @@
-version: '2'
 services:
   erddap:
-    container_name: "erddap_gold_standard"
     image: axiom/docker-erddap:2.23-jdk17-openjdk
     restart: unless-stopped
     ports:
       - "8080:8080"
+    volumes:
+      - ./erddap/conf/robots.txt:/usr/local/tomcat/webapps/ROOT/robots.txt
```

Comment:
Don't you want to map from `%{PWD}/erddap/conf/robots.txt` ?

Reply:
similarly with the next few lines.

Reply:
I don't think so, you can use relative paths in `docker-compose.yml` if they begin with `./` (unless you mean something else!)